hello.java

```diff
@@ -1,5 +1,5 @@
 public class hello{
     public static void main(String[] args) {
-    System.out.println("hello world");    
```

Comment:
why did you delete this code

Reply:
I just think printing the code 5 times is cooler

Reply:
whatever